src/templates/activityTemplate/suggestionsSection/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState, ChangeEvent } from 'react';
+import Suggestions from './suggestions';
+import NewSuggestionForm from './newSuggestionForm';
+import ConfirmationModal from './confirmationModal';
+import { StrapiActivity } from '../../../../graphql-types';
+
+interface SuggestionsSectionProps {
+  activityId: string;
+  data: StrapiActivity;
+}
+
+export interface Error {
+  text: string;
+  err?: any;
+}
+
+const parseActivityId = (id: string) => id.split('_')[1];
```

Comment:
Hey I just noticed that the real id of these is in the `strapiId` field. So you could just add it to the gql-query and use it here and remove this function etc.